test/azure/Expected/AcceptanceTests/XmsErrorResponses/PetOperations.cs

```diff
@@ -0,0 +1,616 @@
+            {
+                _httpRequest.Headers.TryAddWithoutValidation("x-ms-client-request-id", System.Guid.NewGuid().ToString());
+            }
+            if (Client.AcceptLanguage != null)
+            {
+                if (_httpRequest.Headers.Contains("accept-language"))
+                {
+                    _httpRequest.Headers.Remove("accept-language");
+                }
+                _httpRequest.Headers.TryAddWithoutValidation("accept-language", Client.AcceptLanguage);
+            }
+
+
+            if (customHeaders != null)
+            {
+                foreach(var _header in customHeaders)
+                {
+                    if (_httpRequest.Headers.Contains(_header.Key))
+                    {
+                        _httpRequest.Headers.Remove(_header.Key);
+                    }
+                    _httpRequest.Headers.TryAddWithoutValidation(_header.Key, _header.Value);
+                }
+            }
+
+            // Set Credentials
+            if (Client.Credentials != null)
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+                await Client.Credentials.ProcessHttpRequestAsync(_httpRequest, cancellationToken).ConfigureAwait(false);
+            }
+            // Send Request
+            if (_shouldTrace)
+            {
+                ServiceClientTracing.SendRequest(_invocationId, _httpRequest);
+            }
+            cancellationToken.ThrowIfCancellationRequested();
+            _httpResponse = await Client.HttpClient.SendAsync(_httpRequest, cancellationToken).ConfigureAwait(false);
+            if (_shouldTrace)
+            {
+                ServiceClientTracing.ReceiveResponse(_invocationId, _httpResponse);
+            }
+            HttpStatusCode _statusCode = _httpResponse.StatusCode;
+            cancellationToken.ThrowIfCancellationRequested();
+            if ((int)_statusCode != 200 && (int)_statusCode != 202)
+            {
+                try
+                {
+                    switch((int)_statusCode)
+                    {
+                        case 400:
+                            await Handle400ErrorResponseForGetPetById(_httpRequest, _httpResponse);
+                            break;
+                        case 404:
+                            await Handle404ErrorResponseForGetPetById(_httpRequest, _httpResponse);
+                            break;
+                        case 501:
+                            await Handle501ErrorResponseForGetPetById(_httpRequest, _httpResponse);
+                            break;
+                        default:
+                            await HandleDefaultErrorResponseForGetPetById(_httpRequest, _httpResponse, (int)_statusCode);
+                            break;
+                    }
+                }
+                catch (JsonException)
+                {
+                    // Ignore the exception
```

Comment:
when would a JsonException be caught here? seems like this can/should be removed